app/assets/javascripts/directives/datepicker.js

```diff
@@ -1,25 +1,53 @@
-ManageIQ.angular.app.directive('datepickerInit', function () {
+/*
+ * The 'datepicker' angular calendar directive takes care of:
+ * 1. data conversion (String <-> Date) between datepicker input and model
+ * 2. re-configuring datepicker (start date, end date, skip days), whenever
+ *    the model changes.
+ *
+ * Supported attributes:
+ * - miq-cal-date-from: starting date for the datepicker
+ * - miq-cal-date-to: end date for the datepicker
+ * - miq-cal-skip-days: days of the week (array) to disable in the datepicker
+ */
+
+ManageIQ.angular.app.directive('datepicker', function() {
   return {
     require: 'ngModel',
-    link: function (scope, elem, attr, ctrl) {
-      var language = attr.language;
-      scope['form_' + ctrl.$name] = elem;
+    link: function(scope, elem, attr, ctrl) {
+      ctrl.$formatters.push(function(value) {
+        if (value) {
+          return moment(value).utc().format('MM/DD/YYYY');
+        }
+      });
 
-      scope.$watch(attr.ngModel, function() {
-        if((ctrl.$modelValue != undefined)) {
-          scope['form_' + ctrl.$name].datepicker({autoclose: true,
-                                                  format: "mm/dd/yyyy",
-                                                  startDate: ctrl.$modelValue,
-                                                  weekStart: 0,
-                                                  language: language});
-          var component = scope['form_' + ctrl.$name].siblings('[data-toggle="datepicker"]');
-          if (component.length) {
-            component.on('click', function () {
-              scope['form_' + ctrl.$name].trigger('focus');
-            });
-          }
+      ctrl.$parsers.push(function(value) {
+        if (value) {
+          return moment.utc(value, 'MM/DD/YYYY').toDate();
         }
       });
-    }
+
+      scope.$watch(attr.ngModel, function(value) {
+        if(value)
```

Comment:
`if (value) {` please ;)

(IMO the only CC warning we do want to fix here.)

Reply:
Fixed.